client/src/templates/Challenges/redux/completion-epic.js

```diff
@@ -62,16 +62,20 @@ function submitModern(type, state) {
     }
 
     if (type === actionTypes.submitChallenge) {
-      const { id } = challengeMetaSelector(state);
+      const { id, block } = challengeMetaSelector(state);
       const challengeFiles = challengeFilesSelector(state);
       const { username } = userSelector(state);
       const challengeInfo = {
         id,
```

Comment:
```suggestion
        id
```
To make prettier happy.